docs/examples/academy/15. Filtering and Modifying DataFrames.ipynb

```diff
@@ -1355,273 +271,25 @@
-       "      <td>3.028150e+10</td>\n",
-       "      <td>1.000000</td>\n",
-       "      <td>0.044765</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>South America</th>\n",
-       "      <td>2.298574e+07</td>\n",
-       "      <td>1412694.0</td>\n",
-       "      <td>1.541130e+11</td>\n",
-       "      <td>0.500000</td>\n",
-       "      <td>0.055462</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>Southeast Asia</th>\n",
-       "      <td>2.676705e+08</td>\n",
-       "      <td>833667.0</td>\n",
-       "      <td>3.692500e+09</td>\n",
-       "      <td>0.000000</td>\n",
-       "      <td>0.003115</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>Southern Africa</th>\n",
-       "      <td>3.080979e+07</td>\n",
-       "      <td>1769615.0</td>\n",
-       "      <td>8.423000e+09</td>\n",
-       "      <td>1.000000</td>\n",
-       "      <td>0.057437</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>West Africa</th>\n",
-       "      <td>1.958747e+08</td>\n",
-       "      <td>1999885.0</td>\n",
-       "      <td>3.707000e+10</td>\n",
-       "      <td>1.000000</td>\n",
-       "      <td>0.010210</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "                   Population  Oil Production  Proven Reserves  \\\n",
-       "Region                                                           \n",
-       "Central Africa   2.851357e+06        235410.0     1.800000e+09   \n",
-       "Middle East      2.841442e+07       4409331.0     1.319535e+11   \n",
-       "North Africa     2.445348e+07        866523.5     3.028150e+10   \n",
-       "South America    2.298574e+07       1412694.0     1.541130e+11   \n",
-       "Southeast Asia   2.676705e+08        833667.0     3.692500e+09   \n",
-       "Southern Africa  3.080979e+07       1769615.0     8.423000e+09   \n",
-       "West Africa      1.958747e+08       1999885.0     3.707000e+10   \n",
-       "\n",
-       "                 Membership Active  Production per cap  \n",
-       "Region                                                  \n",
-       "Central Africa            1.000000            0.075100  \n",
-       "Middle East               0.833333            0.341946  \n",
-       "North Africa              1.000000            0.044765  \n",
-       "South America             0.500000            0.055462  \n",
-       "Southeast Asia            0.000000            0.003115  \n",
-       "Southern Africa           1.000000            0.057437  \n",
-       "West Africa               1.000000            0.010210  "
-      ]
-     },
-     "execution_count": 12,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "OPEC_df.groupby('Region').mean()"
+    "OPEC_df.groupby(\"Region\").mean(numeric_only=True)"
```

Comment:
This is the only meaningful change in this file - it fixes an issue where the `.mean` method was not working after a Pandas upgrade.